sass/includes/_filters.scss

```diff
@@ -20,7 +22,8 @@
 
     &__fieldset {
         flex-shrink: 0;
-        border: 1px solid $color__black;
+        background-color: $color__white;
```

Comment:
Query: is the border still visible if there is no javascript used on the page?

Reply:
I've tested without js now, and it looks like we still have the white background. I think this is fine - Nick wanted the black outline but if that is tricky to implement lets stick with what we have.

Reply:
The black border isn't in the figma designs so I removed it - I can have another look at this though if it's required, @nicklee please let me know!